server/templates/importer.go

```diff
@@ -35,7 +35,10 @@ func DiscoverTemplateOrder(templateBasePath string) ([][]string, error) {
 			}
 			// only entries with dependencies are added, those without dependencies don't need to be added
 			for _, entryValue := range fileContent {
-				dependenciesGraph.AddDependency(name, entryValue)
+				err = dependenciesGraph.AddDependency(name, entryValue)
+				if err != nil {
+					log.Errorf("Error while adding template dir to dependency graph: %v", err)
```

Comment:
This error should be more verbose, it should at least include the `entryValue` so we know which dependency could not have been added.

Reply:
Added the dependency and parent name to string.